### DISCUSSION TITLE
Logo not embedded in email

### DISCUSSION THREAD

**Discussion by SidSkiba** [InvoiceShelf/InvoiceShelf#195] [Q&A]
I was looking at running InvoiceShelf on WSL 2 in Windows as a localhost. It seems to work alright but if I have a company logo it is not displayed in email. The image reference is expecting to pull that from a public website address. I cannot see any way to delete the logo from the email body in the settings. Is there a way to do this or could the logo omit from email be added as a setting for this self hosted use case?

**Comment by rihards-simanovics** [comment]
You'd need to edit the actual PHP file for the email generation which as far as I aware is not part of the user editable templates.

**Reply by SidSkiba** [reply]
OK thank you. I will try doing that. Hopefully this will be considered for future releases.
